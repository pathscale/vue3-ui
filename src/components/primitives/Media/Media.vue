<script>
export default {
  name: 'VMedia',
  props: {},
}
</script>

<template>
  <article class="media">
    <div class="media-left">
      <slot name="left" />
    </div>
    <div class="media-content">
      <slot name="content" />
    </div>
    <div class="media-right">
      <slot name="right" />
    </div>
  </article>
</template>
