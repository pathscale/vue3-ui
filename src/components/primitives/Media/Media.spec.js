import { shallowMount } from '@vue/test-utils'
import Media from './Media.vue'

let wrapper

describe('Media', () => {
  beforeEach(() => {
    wrapper = shallowMount(Media)
  })

  it('is called', () => {
    expect(wrapper.exists()).toBeTruthy()
  })

  it('render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
