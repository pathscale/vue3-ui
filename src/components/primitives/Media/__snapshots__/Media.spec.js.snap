// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media render correctly 1`] = `
<article class="media">
  <div class="media-left"></div>
  <div class="media-content"></div>
  <div class="media-right"></div>
</article>
`;
